packages/ag-charts-community/src/chart/series/cartesian/barUtil.ts

```diff
@@ -138,3 +141,34 @@ export function prepareAnimationFunctions<T extends AnimatableBarDatum>(
 
     return { toFn, fromFn };
 }
+
+export function getBarDirectionStartingValues(
+    barDirection: ChartAxisDirection,
+    axes: Record<ChartAxisDirection, ChartAxis | undefined>
+) {
+    const isColumnSeries = barDirection === ChartAxisDirection.Y;
+
+    const xAxis = axes[ChartAxisDirection.X];
+    const yAxis = axes[ChartAxisDirection.Y];
+
+    let startingX = Infinity;
+    let startingY = 0;
+
+    if (isColumnSeries) {
+        const isContinuousY = yAxis?.scale instanceof ContinuousScale;
+        if (isContinuousY) {
+            startingY = yAxis.scale.convert(0);
+        } else if (yAxis) {
+            startingY = yAxis.scale.convert(Math.max(...yAxis.range));
```

Comment:
Previously, this would only take into account values where `datum.yValue >= 0`, so it would get the correct line when values are negative as well (or both). Is that implied in the range/conversion? Not sure any of the tests cover this atm.